saleor/tests/fixtures.py

```diff
@@ -440,6 +441,19 @@ def checkout_with_item_and_shipping_method(checkout_with_item, shipping_method):
     return checkout
 
 
+@pytest.fixture
+def checkout_with_item_shipping_method_and_shipping_address(
+    checkout_with_item, shipping_method
+):
+    checkout = checkout_with_item
+    checkout.shipping_method = shipping_method
+    checkout.save()
+    return checkout
+
+    checkout.shipping_address = address.get_copy()
```

Comment:
there is `return` before those lines

Reply:
this fixture was supposed to be deleted. good catch. thx